src/Modules/Login.php

```diff
@@ -143,6 +144,15 @@ public function authenticate( $user = null ) {
 			if ( $user instanceof WP_User ) {
 				$this->authenticated = true;
 
+				/**
+				 * Fires once the user has been authenticated via Google OAuth.
+				 *
+				 * @param WP_User $user WP User object.
+				 *
+				 * @since 1.2.2
```

Comment:
This is the current version and this new action going to release in newer version. So maybe it's `1.2.3` or `1.3.0`?

Reply:
@abhishekfdd